File: app/src/main/java/com/jx/androiddemo/constant/Navigator.java
package com.jx.androiddemo.constant;

import android.content.Context;
import android.content.Intent;

import com.jx.androiddemo.activity.main.MainActivity;

/**
 * Class used to navigate through the application.
 */
public class Navigator {
    /**
     * Goes to MainActivity.
     */
    public static void navigateToMainActivity(Context context) {
        if (context != null) {
            Intent intent = new Intent(context, MainActivity.class);
            context.startActivity(intent);
        }
    }
}

File: app/src/main/java/com/jx/androiddemo/constant/Navigator.kt
package com.jx.androiddemo.constant

import android.content.Context
import android.content.Intent
import com.jx.androiddemo.activity.main.MainActivity

object Navigator {
    fun navigateToMainActivity(context: Context) {
        val intent = Intent(context, MainActivity::class.java)
        context.startActivity(intent)
    }
}